Interface/src/interfaces/HasUniform.java

```diff
@@ -0,0 +1,20 @@
+package interfaces;
+
+public interface HasUniform {
+    boolean UNIFORM = true;
+
+    boolean hasCap();
+
+    boolean getUniform();
+
+    String colorUniform();
+
+    default void uniformDescription() {
+        if (getUniform() == UNIFORM) {
```

Comment:
It looks like getUniform() return hard-code. Please consider better using of interfaces